### PR TITLE
CORTX-28875: RGW: Update port value for rgw.

### PR DESCRIPTION
**Pull request by shriya-deshmukh**
Signed-off-by: Shriya Deshmukh <shriya.deshmukh@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [x] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by cdeshmukh — APPROVED** [review]

File: src/setup/rgw.py [hunk 1]
```diff
@@ -415,13 +415,13 @@ def _update_rgw_config_with_endpoints(conf: MappedConf, endpoints: dict, instanc
...
         Conf.set(Rgw._rgw_conf_idx, f'client.rgw-{instance}>log file', service_instance_log_file)
         # For each instance increase port value by 1.
-        # for eg. for 1st instance. port=8000
-        # for 2nd instance port=8000 + 1
+        # for eg. for 1st instance. port=22751
+        # for 2nd instance port=22751 + 1
         # port = <port> + (instance - 1)
         # TODO: read port value from endpoint url define in cluster.conf
-        port = 8000
+        port = 22751
```
Comment (sachitanands) [inline src/setup/rgw.py:422]: can we move these ports to const.py file as well.. tomorrow it may get changed so better to keep it as constants

Comment (sachitanands) [reply src/setup/rgw.py:422]: yes. agree.. we can skip this comment.

**Review by sachitanands — APPROVED** [review]

**Review by tanujashinde0405 — APPROVED** [review]

**Review by rahul-tripathi-git — APPROVED** [review]
+1

**Review by sachitanands — APPROVED** [review]
looks good

**Pull request merged**